pkgs/applications/audio/dr14_tmeter/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3Packages, pkgs }:
+
+let
+ inherit (python3Packages) buildPythonApplication;
+in buildPythonApplication rec {
+  name = "dr14_tmeter-${version}";
+  version = "1.0.16";
+
+  disabled = !python3Packages.isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "simon-r";
+    repo = "dr14_t.meter";
+    rev = "v${version}";
+    sha256 = "1nfsasi7kx0myxkahbd7rz8796mcf5nsadrsjjpx2kgaaw5nkv1m";
+  };
+
+  propagatedBuildInputs = with pkgs; [
+    python3Packages.numpy flac vorbis-tools ffmpeg faad2 lame
+  ];
+
+  # There are no tests
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "Compute the DR14 of a given audio file according to the procedure described by the Pleasurize Music Foundation";
+    license = stdenv.lib.licenses.gpl3Plus;
```

Comment:
Using `with stdenv.lib` you bring everything into scope that is in `stdenv.lib`. Therefore you can just write `license = licenses.gpl3Plus;`